ToDoLite/DetailViewController.m

```diff
@@ -226,6 +246,18 @@ - (UITableViewCell *)couchTableSource:(CBLUITableSource *)source cellForRowAtInd
                                                     forIndexPath:indexPath];
     CBLQueryRow *row = [source rowAtIndex:indexPath.row];
     Task *task = [Task modelForDocument:row.document];
+
+    int64_t delay = 1.5; // In seconds
```

Comment:
The delay is not necessarily "needed" per se, however I'd argue that it results in better UX. For example, without any delay, *all* text would start off as grey and would (normally, under good network conditions) pretty much immediately turn black. From a user perspective, that would potentially look like a bug, since *all* inputted text would be briefly "flickering".

Having a bit of lag/delay time prevents this "flickering", yet still communicates to the user that something might be up when there indeed is an actual sync issue.